src/test/FuzzerImpl.cpp

```diff
@@ -406,6 +538,37 @@ struct xdr_fuzzer_unpacker
             v % stellar::FuzzUtils::NUMBER_OF_ASSETS_TO_USE);
     }
 
+    template <typename T>
+    typename std::enable_if<
+        std::is_same<stellar::ClaimableBalanceID, T>::value>::type
+    operator()(T& balanceID)
+    {
+        check(1);
+        std::uint8_t v = get_byte();
+        stellar::LedgerKey key;
+        stellar::FuzzUtils::setShortKey(mStoredLedgerKeys, key, v);
+        // If this one byte indexes a stored LedgerKey for a ClaimableBalanceID,
+        // use that; otherwise just use the byte itself as the balance ID.
+        if (key.type() == stellar::CLAIMABLE_BALANCE)
```

Comment:
> In practice, I feel like this generates "bad" balance IDs almost all the time. Any measure on the success rate here?

After a few minutes, 25-26% of keys had taken the `true` path of that `if` -- which I think might not be surprising given that setup is creating 60 claimable balances currently.  Because different setup ledger entries will share some of the same indexes, some of those will have been overwritten by the end of setup, but I can believe that the fuzzer might be leaning a bit towards using the bytes that to map to claimable balances in `ClaimableBalanceID`s after finding that the others exercise fewer new code paths.